cfgov/templates/wagtailadmin/account/password_reset/confirm.html

```diff
@@ -1,19 +1,19 @@
 {% extends "wagtailadmin/admin_base.html" %}
-{% load compress static i18n %}
+{% load static i18n %}
 {% block titletag %}{% trans "Set your new password" %}{% endblock %}
 {% block bodyclass %}login{% endblock %}
 {% load wagtailadmin_tags %}
 
 {% block extra_css %}
     {{ block.super }}
 
-    {% compress css %}
-        <link rel="stylesheet" href="{% static 'wagtailadmin/css/layouts/login.css' %}" type="text/css" />
-    {% endcompress %}
+    <link rel="stylesheet" href="{% static 'wagtailadmin/css/layouts/login.css' %}" type="text/css" />
 {% endblock %}
 
 {% block furniture %}
     <div class="content-wrapper">
+
+    {% if validlink %}
```

Comment:
Did you bring this in from [this commit](https://github.com/wagtail/wagtail/commit/84a2a45d49307b533f16eeb2901b368a65ad7453#diff-a9f218f2a6735e21fd62ef455224ca98)? I think perhaps this isn't available yet, it's still pending in the 2.1 release.